WcaOnRails/lib/auxiliary_data_computation.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+module AuxiliaryDataComputation
+  def self.compute_everything
+    self.compute_best_of_3_in_333bf
+    self.compute_concise_results
+    self.compute_rank_tables
+
+    self.delete_php_cache # Note: this should go away together with the PHP code.
```

Comment:
Oh! I thought the idea was you were deleting the old cachedDatabase.php file, but that's not it at all. The php code has some html caching mechanism, and you're clearing that here.
Sorry for the confusion!